### DISCUSSION TITLE
Problem connecting to server

### DISCUSSION THREAD

**Discussion by SirJaner** [Impostor/Impostor#310] [Q&A]
I have problem, as I want to play with friends, but after i create server no one can connect to it. I on the other hand can connect with this 127.0.0.1 IP if i use it in Impostor Client and i see my connection on server (i didn't do the screenshots). I started from scratch three times and still not working. Port forwarding is the last screen. Pls help!!!

![AmongUsServerImpostor](https://user-images.githubusercontent.com/78509239/106812191-9c9efc00-666f-11eb-9599-004d8cd371c0.png)
![AmongUsInpostorConfigAndClient](https://user-images.githubusercontent.com/78509239/106812194-9d379280-666f-11eb-8777-c25fc7659947.png)
![AmongUs](https://user-images.githubusercontent.com/78509239/106812196-9d379280-666f-11eb-8aea-713ae5df2a82.png)
![Connect Box Portforwarding](https://user-images.githubusercontent.com/78509239/106812758-631ac080-6670-11eb-8f5f-85ed05d3e573.png)

**Comment by stephenplaysamongus** [comment]
I actually had the same problem until I watched this video on how to fix it https://7r6.com/amongushelp

**Reply by higps** [reply]
the link brings you to a malware page

**Comment by higps** [comment]
I'm having the exact issue, were you able to sort it? I'm running other servers on my machine and others can connect to those just fine

**Reply by SirJaner** [reply]
nope, still trying :/ and for me no one can connect

**Comment by LilyArlatto** [comment]
Also having this exact issue, the odd thing is that most port checker websites are also saying that the connection is timing out so I have no idea what's going on.

**Comment by le-show-de-didoub74** [comment]
Hey have you try with 127.0.0.1 

Don't forget to modify you config.json file

If this don't work you server is not correctly configure
